liveblog-embed/pages-manager.service.js

```diff
@@ -201,18 +202,16 @@
              * count the number of new items 
```

Comment:
pls change the text accordingly
also be sure to specify that is updating self.uncountedPosts